src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py

```diff
@@ -489,14 +491,16 @@ def get_chart_mapping_schema(
             new_schema = self.search_schema(deploy_params_dict, schema_data)
         except KeyError as e:
             raise InvalidTemplateError(
-                f"ERROR: There is a problem with your schema or values for the helm package '{helm_package.name}'. \
-                    Please fix this and run the command again."
+                "ERROR: There is a problem with your schema or values for the helm"
+                f" package '{helm_package.name}'.                     Please fix this"
```

Comment:
again

Reply:
Sorry, and thanks for catching. I think that was the merge from main messing things up :-(